PlasMOUL/Avatar/CoopCoordinator.cpp

```diff
@@ -66,3 +66,11 @@ void MOUL::CoopCoordinator::write(DS::Stream* s) const
     s->writeSafeString(m_synchBone);
     s->write<bool>(m_autoStartGuest);
 }
+
+bool MOUL::CoopCoordinator::makeSafeForNet()
+{
+    return (m_hostBrain == nullptr || m_hostBrain->makeSafeForNet())
+        && (m_guestBrain == nullptr || m_guestBrain->makeSafeForNet())
+        // This can only be LinkToAgeMsg.
+        && (m_acceptMsg == nullptr || (m_acceptMsg->type() == ID_LinkToAgeMsg && m_acceptMsg->makeSafeForNet()));
```

Comment:
I think I would prefer to see us only run `makeSafeForNet()` on the `m_acceptMsg` and not constrain on its type. That way, we're free to add more coop coordinators in the future. IIRC, TOC has something bespoke that uses coop coordinator. While they don't use DS, I think that's a good indicator that this is a useful avenue for future features.

Reply:
Sure, can do. I wanted to be safe here and keep DIRTSAND's existing behavior - before my changes here, a similar check was done in `MOUL::AvCoopMsg::makeSafeForNet()`.